nns/namestate.go

```diff
@@ -22,6 +23,10 @@ func (n NameState) ensureNotExpired() {
 
 // checkAdmin panics if script container is not signed by the domain name admin.
 func (n NameState) checkAdmin() {
+	if n.Owner == nil {
+		checkCommittee()
+		return
+	}
```

Comment:
If we don't check (and don't allow) `Admin` for TLDs, then we probably should explicitly fail `setAdmin` for it. There's a `common.CheckOwnerWitness(ns.Owner)` that will fail for an empty owner, but we should probably give more detailed error in this case.

Reply:
fixed